.config/typedoc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "categorizeByGroup": false,
+  "categoryOrder": ["Main Classes", "API Response Types", "API Models", "*"],
+  "entryPoints": ["../src/index.ts"],
+  "excludePrivate": true,
+  "excludeReferences": true,
+  "includeVersion": true,
+  "out": "../docs",
```

Comment:
i like outputting to `docs` but how do we ensure it remains updated, that was a challenge we had with ethereumjs. maybe we run a workflow on commits to main that add a commit to run `npm run docs-build`? will this replace the current github pages? i find the github pages kind of clunky/ugly to use right now